pom.xml

```diff
@@ -405,6 +365,86 @@ SOFTWARE.
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>quliceJava8</id>
+      <activation>
+        <jdk>1.8</jdk>
```

Comment:
@h1alexbel By doing this, you automatically enable  `quliceJava8` or `qulice` plugins.  It means, if I run 
```shell
mvn clean install
```
I will run `quilice` as well. It's suboptimal during development process. More often we don't need to run `qulice` and check offences straight away. I personally run it only before creating a PR. 

So, I suggest to make `qulice` and `quliceJava8` included to `check` profile. [Here](https://stackoverflow.com/questions/943411/can-i-make-one-maven-profile-activate-another) is how you can achieve this.